src/index.ts

```diff
@@ -4899,6 +4899,7 @@ export interface ExperienceContentConfig {
   banner?: BannerExperienceContentConfig
   modal?: ModalExperienceContentConfig
   display?: string
+  secondsUntilAutoHide?: number
```

Comment:
I think this should go in `ExperienceLayoutConfig` instead - there is a ExperienceContentConfig per locale, but only one ExperienceLayoutConfig